ui-tests/src/test/java/org/openmrs/reference/XSSOnPhoneNumberFieldTest.java

```diff
@@ -36,7 +35,6 @@ public void setup() {
     }
 
     @Test
-    @Ignore
```

Comment:
you need to redo this and send through a clean PR without that trailing change i trashed

Reply:
@gracebish did you address the comment above